src/MusicStore.Spa/Compiler/Preprocess/RazorPreCompilation.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
License header is not required for this sample. Can we remove this license header?